app/models/slate_config.py

```diff
@@ -25,10 +25,11 @@ class SlateConfigModel:
     # store loaded slate configs (loaded at app startup)
     SLATE_CONFIGS_BY_ID = {}
 
-    def __init__(self, slate_id: str, display_name: str, description: str, experiments=None):
+    def __init__(self, slate_id: str, display_name: str, description: str, curator_topic_label=None, experiments=None):
         self.id = slate_id
         self.displayName = display_name
         self.description = description
+        self.curator_topic_label = Optional[curator_topic_label]
```

Comment:
The `Optional` modified should go on the `curated_topic_label : Optiona[str]` parameter method, not here.
This doesn't do what you want:
```
>>> from typing import Optional
>>> a = None
>>> b = Optional[a]
>>> type(a)
<class 'NoneType'>
>>> type(b)
<class 'type'>
>>> b
<class 'NoneType'>
>>> a
```